infra/common/src/main/java/org/apache/shardingsphere/infra/datasource/pool/metadata/type/hikari/HikariDataSourcePoolMetaData.java

```diff
@@ -51,6 +51,7 @@ private static void buildDefaultProperties() {
         DEFAULT_PROPS.put("maximumPoolSize", 50);
         DEFAULT_PROPS.put("minimumIdle", 1);
         DEFAULT_PROPS.put("readOnly", false);
+        DEFAULT_PROPS.put("keepaliveTime", 0);
```

Comment:
- There doesn't seem to be a direct unit test class for this class. I just added it because HikariCP 4.0.0 added a new property with a default value of 0.